demos/image_classification_with_string_output/README.md

```diff
@@ -0,0 +1,74 @@
+# String output model demo {#ovms_string_output_model_demo}
+## Overview
+
+This demo demonstrates example deployment of a model with output precision `ov::element::string`. The output text is serialized into corresponding fields in gRPC proto/REST body. This allows the client to consume the text directly and avoid the process of label mapping or detokenization.
+
+### Download and prepare Image Net model using Keras and TensorFlow
+
+```bash
+pip install -r requirements.txt
+python3 download_model.py
+```
+
+### Start the OVMS container:
+```bash
+docker run -d -u $(id -u):$(id -g) -v $(pwd):/workspace -p 9178:9178 openvino/model_server:latest \
+--model_path /workspace/model --model_name image_net --rest_port 9178
+```
+
+## Send request
```

Comment:
Sending the data with http binary data is not easy with the curl command. It think if would be better just to demonstrate python client.